media/src/rooms.js

```diff
@@ -34,11 +34,15 @@ const makeSocket = function(roomId, handleMessage=null) {
     return socket;
 };
 
-const setHost = (socket=null) => {
+const setHost = (socket=null, objects) => {
+    if (!objects) {
+        objects = [];
+    }
     if (socket) {
         socket.send(JSON.stringify({
             message: {
-                setHost: true
+                setHost: true,
+                publishScene: objects
```

Comment:
When the `publishScene` attribute is present, the if clause on line 97 of `consumers.py` will actually get executed, and this will be considered a `publishScene` event. This causes a bug where the object transfer works, but I'm not actually the host of the room I just made.

To prevent bugs like this, let's just keep a single message attribute on the top level here, and we can restructure it like this:
```
        socket.send(JSON.stringify({
            message: {
                setHost: objects
```